### PR TITLE
Adding comment

### PR DESCRIPTION
**Pull request by ymassad**

### CODE REVIEW

File: _posts/2020-02-24-discerning-and-maintaining-purity.html [hunk 1]
```diff
@@ -296,4 +296,22 @@ <h3 id="67101ecee7b94a849b24126ac01851fe">
...
+<div id="comments">
+	<hr>
+	<h2 id="comments-header">
+		Comments
+	</h2>
+	<div class="comment" id="1925963B8A194CF9BC44E0EB59E92C9C">
+		<div class="comment-author"><a href="http://criticalsoftwareblog.com">Yacoub Massad</a></div>
+		<div class="comment-content">
+			<p>
+				You might be interested in taking a look at [PurityAnalyzer](https://github.com/ymassad/PurityAnalyzer). An open source roslyn-based analyzer for C# that I started developing to help maintain pure C# code.
```
Comment (a1300) [inline _posts/2020-02-24-discerning-and-maintaining-purity.html]: @ymassad this links isn't valid

Comment (ymassad) [reply _posts/2020-02-24-discerning-and-maintaining-purity.html]: The link itself is valid. I thought I could use Markdown. Will change to an an element.

Comment (a1300) [reply _posts/2020-02-24-discerning-and-maintaining-purity.html]: @ymassad yes, I meant the link itself is not rendered 🙂

Comment (ymassad) [reply _posts/2020-02-24-discerning-and-maintaining-purity.html]: I have edited it to fix it. Thanks.

**Comment by ploeh** [comment]
Thank you for writing 👍 I've left [a response on the page](https://blog.ploeh.dk/2020/02/24/discerning-and-maintaining-purity/#50049d1906e04e1ab3811765ca5c3156).

**Pull request merged**